src/app/clients/clients-routing.module.ts

```diff
@@ -83,8 +83,7 @@ const routes: Routes = [
             data: { title: extract('General'), breadcrumb: 'General', routeParamBreadcrumb: false },
             resolve: {
               clientAccountsData: ClientAccountsResolver,
-              clientChargesData: ClientChargesResolver,
-              clientSummary: ClientSummaryResolver
```

Comment:
Why removing the clientSummary?

Reply:
Usually there is an error with that report that blocks other steps